README.md

```diff
@@ -5,7 +5,7 @@ jQuery.i18n is a jQuery based Javascript internationalization library. It helps
 
 This is a project by Wikimedia foundation's [Language Engineering team](http://wikimediafoundation.org/wiki/Language_Engineering_team) and used in some of the Wikimedia Foundation projects like Universal Language Selector.
 
-The jquery.i18n library uses a json based localization file format and it is now used as localization file format for  MediaWiki.
+The jquery.i18n library uses a json based localization file format, "banana", which is used as the localization file format for  MediaWiki and other projects.
```

Comment:
Yeah, the README has a few other tweaks that'd be good; I stopped myself from making too many before people had seen it yet. :-)